eslint-config-feedzai-react/rules/react-hooks.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * React Hooks Rules
+ *
+ * @author João César (joao.cesar@feedzai.com)
+ * @since 1.0.0
+ */
+
+module.exports = {
+    plugins: ["react-hooks"],
+
+    rules: {
+
+        // Validates React Hooks rules
+        "react-hooks/rules-of-hooks": "error",
+
+        // Rule that verifies the list of dependencies in methods like useEffect, useCallback, etc
+        // https://github.com/facebook/react/issues/14920
+        "react-hooks/exhaustive-deps": "warn"
```

Comment:
Why not put this as error?

Reply:
Just added the default: https://www.npmjs.com/package/eslint-plugin-react-hooks. But i agree with changing it to error.